Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -11,23 +11,23 @@ function vtkMouseRangeManipulator(publicAPI, model) {
 
   // Keep track of delta that is below the value
   // of one step to progressively increment it
-  model.incrementalDelta = {};
+  model.incrementalDelta = new Map();
```

Comment:
Can we make the Map as private so it is not stored on the model? That way that won't break the state serialization process.